source/ocsp-support/ocsp-support.rst

```diff
@@ -452,6 +491,17 @@ No. Although `RFC 5019, The Lightweight Online Certificate Status Protocol
 allows for the configuration of a tolerance period for the acceptance of OCSP
 responses after ``nextUpdate``, this spec is not adhering to that RFC.
 
+Why was the decision made to allow OCSP to be enabled/disabled via a URI option?
+--------------------------------------------------------------------------------
+We initially hoped that we would be able to not expose any options
+specifically related to OCSP to the user, in accordance with the
+“\ `No Knobs” drivers mantra
+<https://github.com/mongodb/specifications#no-knobs>`__. However, we
+later decided that users may benefit from having the ability to toggle
+OCSP off when applications are deployed behind restrictive firewall
```

Comment:
OCSP endpoint checking. Be explicit about that. We're not allowing users to disable all of OCSP.

Reply:
Ah good call: fixed!